packages/coordinator/scripts/uploadRoundMetadata.ts

```diff
@@ -82,65 +100,46 @@ export async function collectMetadata(): Promise<RoundMetadata> {
     });
 
   const askRegistrationEndTime = () =>
-    new Promise<Date>((resolve, reject) => {
-      rl.question(
-        "When would you like to end the registration for applications? (Please respond in the format {Year}/{Month}/{Day} {Hour}:{Minute}:{Second} in UTC time) ",
-        (answer) => {
-          const valid = isValidDate(answer);
-
-          if (!valid) {
-            reject(new Error("Please answer in valid format."));
-          }
-
-          // eslint-disable-next-line no-console
-          console.log(`Your application registration end date for this round is: ${answer}`);
-          resolve(new Date(answer));
-        },
-      );
-    });
-
-  const askVotingStartTime = () =>
-    new Promise<Date>((resolve, reject) => {
+    new Promise<number>((resolve, reject) => {
       rl.question(
-        "When would you like to start the voting for this round? (Please respond in the format {Year}/{Month}/{Day} {Hour}:{Minute}:{Second} in UTC time) ",
+        "How long would you like for the application registration period to be?? (Please respond in seconds) ",
```

Comment:
I think we need to change the wording here to something like: "Please specify the duration of application registration in seconds"